coral/src/app/features/topics/details/schema/TopicDetailsSchema.test.tsx

```diff
@@ -93,4 +132,60 @@ describe("TopicDetailsSchema", () => {
 
     expect(previewEditor).toBeVisible();
   });
+
+  it("shows a Delete button", () => {
```

Comment:
We're not doing "delete" at the moment, because that is not available (see epic https://github.com/aiven/klaw/issues/1297)

Reply:
[ad706cf](https://github.com/aiven/klaw/pull/1344/commits/ad706cf4d317f8c96701c31a52af4af195da3f98)